community/csv/src/main/java/org/neo4j/csv/reader/Configuration.java

```diff
@@ -25,6 +25,13 @@
 public interface Configuration
 {
     /**
+     * TODO: Default value is false here, but it is our intention to flip this to true at some point
+     * because of how it better complies with common expectancy of behavior. It may be least disruptive
+     * to do this when changing major version of the product.
+     */
+    boolean DEFAULT_LEGACY_STYLE_QUOTING = true;
```

Comment:
The comment says this is currently `false`, but this line claims otherwise. Which version conforms the the RFC, the legacy style or the not legacy style?